packages/paste-website/src/pages/components/code-block/index.mdx

```diff
@@ -0,0 +1,186 @@
+slug: /components/code-block/
+---
+
+import {graphql} from 'gatsby';
+import {CodeBlock, CodeBlockGroup} from '@twilio-paste/code-block';
+import Changelog from '@twilio-paste/code-block/CHANGELOG.md';
+
+import {SidebarCategoryRoutes} from '../../../constants';
+import {multiLine} from '../../../component-examples/CodeBlockExamples.ts';
+
+export const pageQuery = graphql`
+  {
+    allPasteComponent(filter: {name: {eq: "@twilio-paste/code-block"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(frontmatter: {slug: {eq: "/components/code-block/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+        description
+      }
+      headings {
+        depth
+        value
+      }
+    }
+    allAirtable(filter: {data: {Feature: {eq: "Code Block"}}}) {
+      edges {
+        node {
+          data {
+            Documentation
+            Figma
+            Design_committee_review
+            Engineer_committee_review
+            Code
+            status
+          }
+        }
+      }
+    }
+  }
+`;
+
+<NormalizedComponentHeader
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/code-block"
+  storybookUrl="/?path=/story/components-code-block--default"
+  data={props.data}
+  description={props.data.mdx.frontmatter.description}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+## Guidelines
+
+### About Code Block
+
+The Snippet component improves readability of embedded code samples with syntax highlighting and automatic line numbering. All provided languages support both a light and dark theme.
+
+Multiple Snippets may be combined with the SnippetGroup component. A common use case for SnippetGroups is providing a code sample in multiple languages.
+
+### Accessibility
+
+Both Snippet and SnippetGroup are fully keyboard accessible. Here are some tips to ensure a positive user experience for all users:
+
+- Provide a descriptive label such as the current filename or step in a process to clarify the purpose of the Snippet.
+- Enable long line wrapping whenever possible for a better reading experience.
+
+## Examples
+
+### Single line
+
+Single line Snippets exclude the label bar and line numbers. It is automatically detected but will default to the multi line layout if line wrapping is enabled.
+
+<LivePreview scope={{CodeBlock}} language="jsx">
+  {`<CodeBlock language='shell-session'>
+  ${`$ curl -G https://api.twilio.com/2010-04-01/Accounts -u '[YOUR ACCOUNT SID]:[YOUR AUTH TOKEN]'`}
+</CodeBlock>`}
+</LivePreview>
+
+### Multi line
+
+A multi line snippet displays a label and line numbers by default. If no label is provided the snippet language will be used as a fallback.
+
+<LivePreview scope={{CodeBlock}} language="jsx" noInline>
+  {multiLine}
+</LivePreview>
+
+### Grouped
```

Comment:
Note: I'm waiting to add the grouped example until I figure out the multi line example.